jobsdb/jobsdb.go

```diff
@@ -860,7 +860,7 @@ func (jd *HandleT) init() {
 				jd.runAlwaysChangesets(templateData)
 
 				// finally refresh the dataset list to make sure [datasetList] field is populated
-				_, err := jd.refreshDSList(l)
+				err := jd.doRefreshDSRangeList(l)
```

Comment:
**Note:** this was actually a bug, but not a critical one, since the ds range list in gateway pods is not of much importance